cpp/velox/compute/WholeStageResultIterator.cc

```diff
@@ -237,8 +237,8 @@ void WholeStageResultIterator::collectMetrics() {
     const auto& nodeId = orderedNodeIds_[idx];
     if (planStats.find(nodeId) == planStats.end()) {
       if (omittedNodeIds_.find(nodeId) == omittedNodeIds_.end()) {
-        DEBUG_OUT << "Not found node id: " << nodeId << std::endl;
-        DEBUG_OUT << "Plan Node: " << std::endl << veloxPlan_->toString(true, true) << std::endl;
+        LOG(WARNING) << "Not found node id: " << nodeId;
+        LOG(WARNING) << "Plan Node: " << std::endl << veloxPlan_->toString(true, true);
```

Comment:
> replace DEBUG_OUT with DLOG

Seems not matched with this description. Any reason here?

Reply:
Changing the log type to warning in the exception block does make sense to me.